From 9ee5f00681285e52deb215d183e8ba9677738a8d Mon Sep 17 00:00:00 2001
From: Chris Meyer <cmeyer1969@gmail.com>
Date: Fri, 7 Oct 2022 10:03:44 -0700
Subject: [PATCH] Find windeployqt in proper location.

---
 launcher/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/launcher/CMakeLists.txt b/launcher/CMakeLists.txt
index b66fa49..5e63de6 100644
--- a/launcher/CMakeLists.txt
+++ b/launcher/CMakeLists.txt
@@ -169,7 +169,10 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     get_filename_component( _dir ${uic_location} DIRECTORY)
     set(deployqt "${_dir}/windeployqt.exe")
     if(NOT EXISTS ${deployqt})
-      message(FATAL_ERROR "Failed to locate deployqt executable: [${deployqt}]")
+      set(deployqt "${_dir}/bin/windeployqt.exe")
+      if(NOT EXISTS ${deployqt})
+        message(FATAL_ERROR "Failed to locate deployqt executable: [${deployqt}]")
+      endif()
     endif()
 
     add_custom_command(TARGET ${APP_NAME}
-- 
2.37.3

